src/ipyaladin/widget.py

```diff
@@ -928,6 +1009,88 @@ def selection(self, selection_type: str = "rectangle") -> None:
             raise ValueError("selection_type must be 'circle' or 'rectangle'")
         self.send({"event_name": "trigger_selection", "selection_type": selection_type})
 
+    def select_region(self, region: SupportedSelectionRegion) -> None:
+        """Triggers Aladin Lite to select a given astropy region.
+
+        Parameters
+        ----------
+        __________
+        region: `~regions.CircleSkyRegion`, `~regions.PolygonSkyRegion`, or a
+        `~regions.RectangleSkyRegion`
+            The selection region to add in Aladin Lite. It can be given as a supported
+            region from the `regions package <https://astropy-regions.readthedocs.io>`_.
+        """
+        if Region is None:
+            raise ModuleNotFoundError(
+                "To read regions objects, you need to install the regions library with "
+                "'pip install regions'."
+            )
+
+        event_name = "trigger_select_region"
+        if type(region) is CircleSkyRegion:
+            ra = region.center.ra.value
+            dec = region.center.dec.value
+            radius = region.radius.value
+            self.send(
+                {
+                    "event_name": event_name,
+                    "selection_type": "circle",
+                    "startCoo": {
+                        "ra": ra,
+                        "dec": dec,
+                    },
+                    "endCoo": {"ra": ra + radius, "dec": dec},
```

Comment:
Moving along ra (longitude) here won't give the right size circle unless `dec == 0` (the equator).  As you get closer to the poles, a "degree" of ra gets smaller and smaller in actual distance.

This approach would be correct here if it kept the ra constant and moved along an ra line since dec degrees are actual degrees and don't change in magnitude depending on where they are on the sphere.  In that case you would still need to account for wrap-around effects (`dec + radius > 90`).

Reply:
I might be misunderstanding what you're describing here, I've done a fair amount of tests and not run into any issues with this either at the poles or near the equator. This might be something that you have to show to me visually for me to understand.